app/controllers/api/storage.php

```diff
@@ -508,9 +508,16 @@
                 if (empty($data)) {
                     $data = $deviceFiles->read($path);
                 }
-                $key = App::getEnv('_APP_OPENSSL_KEY_V1');
+
+                $fileSecret = \bin2hex(OpenSSL::randomPseudoBytes(128));
```

Comment:
Same, can we abstract the secret generation?

Reply:
will do.